code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -394,6 +394,12 @@
 	var/fortify = 0
 	var/crest_defense = 0
 
+	// Baneling vars
+	/// How long smoke lasts for when baneling explodes
+	var/baneling_smoke_duration = 4
+	/// Reference to banelings pod
+	var/obj/structure/xeno/baneling_pod/pod_ref
```

Comment:
Come to think of it, is there any reason this needs to be on the mob and not on the action?